optuna/pruners/hyperband.py

```diff
@@ -49,22 +49,40 @@ class HyperbandPruner(BasePruner):
         Thus, for example, if ``HyperbandPruner`` has :math:`4` pruners in it,
         at least :math:`4 \\times 10` pruners are consumed for startup.
 
+    Example:
+
+        We minimize an objective function with Hyperband pruning algorithm.
+
+        .. testcode::
+
+            from optuna import create_study
+            from optuna.pruners import HyperbandPruner
+
+            def objective(trial):
+                x = trial.suggest_uniform('x', -1, 1)
+                return x ** 2
+
+            study = create_study(pruner=HyperbandPruner())
+            study.optimize(objective)
```

Comment:
Please add `n_trials` (or `timeout`) to stop optimization.
```suggestion
            study.optimize(objective, n_trials=20)
```